lib/src/view/ui/heart_list.dart

```diff
@@ -0,0 +1,170 @@
+
+  _fetchHeartList() {
+    int userId = UserProvider.userId;
+    _heartList = heartService.getHeartsByUser(userId);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: CustomAppBar(titleText: '관심 상품 목록'),
+      body: Center(
+          child: Container(width: size.getSize(335.0), child: _buildBody())),
+    );
+  }
+
+  Widget _buildBody() {
+    return FutureBuilder<HeartListModel>(
+        future: _heartList,
+        builder: (context, snapshot) {
+          if (snapshot.hasData) {
+            _heartListContent = snapshot.data?.dataList;
+            return Column(
+              children: [
+                _buildCheckButton(),
+                Expanded(
+                    child: ListView.builder(
+                  itemCount: _heartListContent?.length,
+                  itemBuilder: (context, index) => _buildItem(index),
+                ))
+              ],
+            );
+          } else if (snapshot.hasError) {
+            return Text("${snapshot.error}");
+          } else {
+            return const Center(
+              child: CupertinoActivityIndicator(
+                radius: 20.0,
+              ),
+            );
+          }
+        });
+  }
+
+  Widget _buildItem(int index) {
+    Product? product = _heartListContent?.elementAt(index).product;
+    return Column(children: [
+      SizedBox(
+        height: size.getSize(8.0),
+      ),
+      Row(
+        children: [
+          ClipRRect(
+            borderRadius: BorderRadius.circular(5.0),
+            child: Image.asset(
+              'assets/images/test.jpeg',
+              width: size.getSize(65.0),
+              height: size.getSize(65.0),
+            ),
+          ),
+          SizedBox(
+            width: size.getSize(12.0),
+          ),
+          Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              B14Text(text: product?.productName),
+              RichText(
+                  text: TextSpan(children: [
+                const TextSpan(
+                    text: '원가 ',
+                    style: TextStyle(
+                        fontSize: 14.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.black)),
+                TextSpan(
+                    text: '${product?.productCost}원',
+                    style: const TextStyle(color: Colors.black))
+              ])),
+              _statusBadge(product),
+            ],
+          ),
+          Expanded(child: Container()),
+          IconButton(
+              onPressed: () {},
+              icon: Icon(
+                Icons.favorite,
+                size: size.getSize(25.0),
+                color: green,
+              ))
+        ],
+      ),
+      SizedBox(
+        height: size.getSize(8.0),
+      ),
+      const Divider(
+        height: 1.0,
+        color: grey153,
+      )
```

Comment:
```suggestion
      Space(height: 8.0),
      CustomDivider()
```
이런식으로 Space나 CustomDivider 사용하면 더 편하게 개발하실 수 있을 것 같아용! 
CustomDivider 만들어놓고 마이페이지화면에서는 저도 사용을 안했네여...?? 추후 수정하겠습니당

Reply:
네 알겠습니당 대체 가능한부분들 수정해서 낼 풀 하겠습니다~!